src/dune

```diff
@@ -26,7 +26,27 @@
          stubs-ghash
          stubs-poly1305
          stubs-siphash
-         stubs-blake3))
+         stubs-blake3)
+  (extra_deps
```

Comment:
you can check that these are now included in the build but not compiled, e.g. with

```
$ dune build -p cryptokit --display=short

```